src_c/simd_blitters_avx2.c

```diff
@@ -4,6 +4,14 @@
 #include <immintrin.h>
 #endif /* defined(HAVE_IMMINTRIN_H) && !defined(SDL_DISABLE_IMMINTRIN_H) */
 
+#define RAISE_AVX2_RUNTIME_SSE2_COMPILED_WARNING()     \
```

Comment:
ps. I'm not sure the consequences of this warning. I think warnings only get printed once in modern pythons (but I can't remember if it's 3.6+). We can remove the warning later if it turns out to spam the console.